### PR TITLE
allow >1 store in local and remote clusters

### LINKED ISSUE #3471: test clusters should (sometimes) run with more than one store
both on local, but in particular on remote clusters.

### PR DESCRIPTION
**Pull request by tbg**
and defaults to two stores.

Fixes #3471.

[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/3582)

### CODE REVIEW

File: acceptance/cluster/localcluster.go [hunk 1]
```diff
@@ -97,25 +97,25 @@ type Event struct {
...
-	CertsDir       string
-	monitorStopper chan struct{}
-	logDir         string
-	ForceLogging   bool // Forces logging to disk on a per test basis
+	client              dockerclient.Client
+	stopper             chan struct{}
+	mu                  sync.Mutex // Protects the fields below
+	dns                 *Container
+	vols                *Container
+	numLocal, numStores int
```
Comment (mberhault) [inline acceptance/cluster/localcluster.go]: nit: I think we generally give each struct field its own line.

**Comment by mberhault** [comment]
LGTM

File: acceptance/cluster/localcluster.go (hunk 1 above)
Comment (tschottdorf) [inline acceptance/cluster/localcluster.go]: done.

**Comment by tschottdorf** [comment]
something borked here (tries to bootstrap the wrong thing on docker); I'm looking.

**Comment by tschottdorf** [comment]
Set the default to one store. Will figure out separately what breaks when there are more.

**Pull request merged**